src/compas_eve/ghpython/components/Ce_BackgroundTask/code.py

```diff
@@ -5,57 +5,57 @@
 
```

Comment:
`Ce` on purpose in file name?

Reply:
Yep! It's `C(ompas)E(ve)`, I think we use this convention everywhere, don't we? https://github.com/compas-dev/compas_fab/tree/main/src/compas_fab/ghpython/components :P

Reply:
damn I've been doing these uppercase all along. my bad..

Reply:
ahh... well, we never really wrote defined if it has to be upper or lower case :)